fedimint-bin-tests/src/main.rs

```diff
@@ -1047,6 +1047,35 @@ struct DevFed {
     esplora: Esplora,
 }
 
+#[allow(unused)]
+struct BaseDaemons {
```

Comment:
Maybe a better name might be `ExternalDaemons` or something. `fedimintd` is really the "base" one in our system, but grouping these out in principle sounds great 👍